SlowCheetah.VisualStudio/Utilities/PackageUtilities.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using EnvDTE;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace SlowCheetah.VisualStudio
+{
+    public class PackageUtilities
+    {
+        private static readonly List<string> ExcludedExtensions = new List<string>(new string[] { ".htm", ".html", ".cs.vb", ".txt", ".jpg", ".png", ".ico", ".aspx", ".snk", ".dll", ".pdb", ".settings" });
+
+        /// <summary>
+        /// Given a string which contains a list of extensions separated by semi-colons, this function sorts through the list
+        /// throwing out any blank or empty items, and then makes sure that the extensions start with a ".".
+        /// The validated list is returned.
+        /// </summary>
+        /// <param name="items">A list of extensions separated by semi-colans. The items may be prefixed with the ".", but don't have to be.</param>
+        /// <returns>The validated and culled list of extensions. The returned items all begin with a ".".</returns>
+        public static string[] GetExtensionList(string items)
+        {
+            const string extensionPrefix = ".";
+            List<string> list = new List<string>();
+            if (!string.IsNullOrWhiteSpace(items))
+            {
+                string[] extensions = items.Split(';');
+                foreach (string extension in extensions)
+                {
+                    string extensionValue = string.IsNullOrWhiteSpace(extension) ? string.Empty : extension.Trim();
+                    if (extensionValue != string.Empty)
+                    {
+                        if (!extensionValue.StartsWith(extensionPrefix))
+                        {
+                            extensionValue = extensionPrefix + extensionValue;
+                        }
+
+                        list.Add(extensionValue);
+                    }
+                }
+            }
+            return list.ToArray();
+        }
+
+        /// <summary>
+        /// Gets a list of unsupported transformation extensions,
+        /// each starting with a ".".
+        /// </summary>
+        /// <returns>The list of extensions</returns>
+        public static List<string> GetExcludedExtensions()
```

Comment:
What is the point of this? `ExcludedExtensions` is already `readonly`, so you can make that public. You should also probably type it as `IReadOnlyCollection<string>` or even `IEnumerable<string>` if no one needs access to `Count`.